### ISSUE TITLE
Use `enter` instead of `v` to paste directly from menu

### ISSUE THREAD

**Issue by jandresfg** [Tudmotu/gnome-shell-extension-clipboard-indicator#412]
I couldn't find the option to override `v` to paste directly from menu, could it be added?

**Comment by jasiralavibiztripz** [comment]
This would be really convenient option to have.
Thanks for the great extension.

**Comment by Tudmotu** [comment]
Yeah, I would generally like to have an option in the settings to customize the shortcuts for all the keyboard actions (paste, pin, delete).

If anyone would like to implement that, I will happily accept a PR.
